src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs

```diff
@@ -146,7 +146,8 @@ private bool IsAnalyzerSuppressedForTree(DiagnosticAnalyzer analyzer, SyntaxTree
             AnalyzerManager analyzerManager,
             CancellationToken cancellationToken = default)
         {
-            onAnalyzerException ??= (ex, analyzer, diagnostic) => { };
+            Action<Exception, DiagnosticAnalyzer, Diagnostic, CancellationToken> newOnAnalyzerException =
+                (ex, analyzer, diagnostic, _) => onAnalyzerException?.Invoke(ex, analyzer, diagnostic);
```

Comment:
📝 I can't find any usage of this public API so far.

Reply:
💡 Consider updating this method to use the new delegate type (with cancellation token), and update the caller to wrap the delegate in one that ignores cancellation.

Reply:
> 📝 I can't find any usage of the only caller (public API `CompilationWithAnalyzers.IsDiagnosticAnalyzerSuppressed`).

Yes, it seems it might have been used in the past from the IDE, but is no longer being used.